web/packages/teleport/src/components/ToolTipNoPermBadge/ToolTipNoPermBadge.tsx

```diff
@@ -16,15 +16,19 @@
 
 import React, { useState } from 'react';
 import styled from 'styled-components';
-import { Text, Popover, Box } from 'design';
+import { Popover, Box } from 'design';
 
 type Props = {
   borderRadius?: number;
+  tooltipText?: TooltipText;
```

Comment:
tooltip text sounded like the text for popover
```suggestion
  badgeTitle?: BadgeTitle;
```

Reply:
Updated this and below comments in https://github.com/gravitational/teleport/pull/31022/commits/3a71196fe14e220df8686c0085045d1ed19cd07a